tests/integration/requests/test_submit_multisigned.py

```diff
@@ -0,0 +1,118 @@
+from unittest import TestCase
+
+from tests.integration.it_utils import JSON_RPC_CLIENT
+from tests.integration.reusable_values import FEE, WALLET
+from xrpl.core.binarycodec import encode_for_multisigning
+from xrpl.core.keypairs import sign
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.requests import SubmitMultisigned
+from xrpl.models.transactions import (
+    Signer,
+    SignerEntry,
+    SignerListSet,
+    TrustSet,
+    TrustSetFlag,
+)
+from xrpl.transaction import (
+    send_reliable_submission,
+    transaction_json_to_binary_codec_form,
+)
+from xrpl.wallet import Wallet
+
+#
+# Set up signer list
+FIRST_SIGNER = Wallet.create()
+SECOND_SIGNER = Wallet.create()
+LIST_SET_TX = send_reliable_submission(
+    SignerListSet(
+        account=WALLET.classic_address,
+        sequence=WALLET.next_sequence_num,
+        last_ledger_sequence=WALLET.next_sequence_num + 10,
+        fee=FEE,
+        signer_quorum=2,
+        signer_entries=[
+            SignerEntry(
+                account=FIRST_SIGNER.classic_address,
+                signer_weight=1,
+            ),
+            SignerEntry(
+                account=SECOND_SIGNER.classic_address,
+                signer_weight=1,
+            ),
+        ],
+    ),
+    WALLET,
+    JSON_RPC_CLIENT,
+)
+
+
+class TestSubmitMultisigned(TestCase):
+    def test_basic_functionality(self):
+        #
+        # Perform multisign
+        #
+        # NOTE: If you need to use xrpl-py for multisigning, please create an issue on
+        # the repo. We'd like to gauge interest in higher level multisigning
+        # functionality.
+        issuer = Wallet.create()
+        tx = TrustSet(
+            account=WALLET.classic_address,
+            sequence=WALLET.next_sequence_num,
+            fee=FEE,
+            flags=TrustSetFlag.TF_SET_NO_RIPPLE,
+            limit_amount=IssuedCurrencyAmount(
+                issuer=issuer.classic_address,
+                currency="USD",
+                value="10",
+            ),
+        )
+        tx_json = transaction_json_to_binary_codec_form(tx.to_dict())
+        first_sig = sign(
+            bytes.fromhex(
+                encode_for_multisigning(
+                    tx_json,
+                    FIRST_SIGNER.classic_address,
+                )
+            ),
+            FIRST_SIGNER.priv_key,
+        )
```

Comment:
IMO `.encode()` should be an attribute on BaseModel. On transactions it could even decide whether it's for (regular) signing or not based on whether there's already a (filled out) `TxnSignature` field. For multi-signing, you might have to pass a specific option or something.